server/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore the default AAS storage directory
+storage/
+
+# Ignore PyCharm IDE configuration
+.idea
```

Comment:
This is already taken care by the toplevel `.gitignore. 

I think we should incorporate this file the the toplevel `.gitignore`. While I agree that "project"-level `.gitignore`s make sense, I am currently unsure what things one has to pay attention for that. Therefore, I say we put it all in one and separate later, if needed.